lib/tools/node-modules/node-modules-dest-copy.ts

```diff
@@ -10,33 +10,21 @@ export interface ILocalDependencyData extends IDependencyData {
 export class TnsModulesCopy {
 	constructor(
 		private outputRoot: string,
-		private $options: IOptions,
 		private $fs: IFileSystem,
 		private $pluginsService: IPluginsService
 	) {
 	}
 
-	public copyModules(dependencies: IDependencyData[], platform: string): void {
-		for (const entry in dependencies) {
-			const dependency = dependencies[entry];
+	public copyModules(opts: { dependencies: IDependencyData[], release: boolean }): void {
+		const filePatternsToDelete = opts.release ? "**/*.ts" : "**/*.d.ts";
```

Comment:
Should we exclude `.scss` files here as well?

Reply:
I was thinking the same, but it is not a common practice to publish packages with `.scss` ,while all plugins have `.d.ts`. We need a more generic way to handle this and let the user decides what to exclude.